<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientJoinWindow</class>
 <widget class="QMainWindow" name="ClientJoinWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClientJoinWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_connect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>114</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_port">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>80</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ip_address">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>46</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Server address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nickname">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>12</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="maxLength">
     <number>31</number>
    </property>
    <property name="placeholderText">
     <string>Nickname</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_nickname</tabstop>
  <tabstop>le_ip_address</tabstop>
  <tabstop>le_port</tabstop>
  <tabstop>btn_connect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
